fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/models/ResourceFactoryImplTest.java

```diff
@@ -121,32 +153,41 @@ public void setup() throws Exception {
         when(psSession.getHeaders(eq(fedoraId), nullable(Instant.class))).thenReturn(resourceHeaders);
     }
 
+    @After
+    public void cleanUp() {
+        when(mockResource.getId()).thenReturn(FEDORA_ID_PREFIX);
```

Comment:
Seems pretty weird to me to mock the resource to return different identifiers depending on if getId or getFedoraId is called. Is that necessary?

Reply:
So `getFedoraId()` returns the `FedoraId` class, but because we still use strings for some things, `getId()` just calls the FedoraId's `getFullId()` method. If we have no use for strings then perhaps this could be removed.

Reply:
Ohhhhhh, yeah with all the changes I can't remember why I did that. I'll revisit it.